src/libslic3r/PrintConfig.hpp

```diff
@@ -911,6 +911,8 @@ PRINT_CONFIG_CLASS_DERIVED_DEFINE(
     ((ConfigOptionFloat,              initial_layer_acceleration))
     ((ConfigOptionFloatOrPercent,     bridge_acceleration))
     ((ConfigOptionFloat,              travel_acceleration))
+    ((ConfigOptionFloat,              sparse_infill_acceleration))
```

Comment:
Can you make it a ConfigOptionFloatOrPercent type, like bridge_acceleration?
The `ratio_over` can be set to  `default_acceleration`.
So that the changes won't disrupt profiles that don't have these settings.